docs/faq.rst

```diff
@@ -245,6 +245,13 @@ What commit of Read the Docs is in production?
 We deploy readthedocs.org from the `rel` branch in our GitHub repository. You can see the latest commits that have been deployed by looking on GitHub: https://github.com/readthedocs/readthedocs.org/commits/rel
 
 
+How can I remove stale results from the search?
+-----------------------------------------------
+
+If the search results contains stale/outdated results,
+you can update the search index by :doc:`wiping the build environment <guides/wipe-environment>` and then rebuilding your docs.
```

Comment:
Not sure about this. I mean, if that is happening I would like that users report that to us, since it is a bug from our side. We could remove this or add something like if this is happening very often, report it.

@davidfischer what do you think?

Reply:
@stsewd 
I think this is a bug from Sphinx, if a `.rst` file is deleted, then the `.html` file for that shouldn't be created or deleted in the next build.

I think it would be better if a user tries wiping the environment and rebuilding the docs, if that doesn't work -- then report to us.

Reply:
Well, as we move toward each build being totally fresh and not relying on a previous environment, this problem should naturally go away. It is possible that we need to detect when files *aren't* generated as part of the build and remove them. Even with this change, that won't be done exactly.